python/sglang/srt/models/base.py

```diff
@@ -0,0 +1,15 @@
+from torch import nn
+
+
+class BaseCausalLM(nn.Module):
```

Comment:
If we only need to add these two functions, we do not need to create a new base class here.

We can define two functions
 `def export_static_state(module)` and `def import_static_state(module, static_params)`

Reply:
I was considering about customizability (e.g. a model may have different opinion about this), but yes this seems not needed at least today.